ext/hal/nxp/mcux/Kconfig

```diff
@@ -108,4 +108,14 @@ config HAS_MCUX_USB_EHCI
 	help
 	  Set if the USB controller EHCI module is present in the SoC.
 
+config HAS_MCUX_USDHC1
+	bool
+	help
+	  Set if the USDHC instance 1 module is present in the SoC.
+
+config HAS_MCUX_USDHC2
+	bool
+	help
+	  Set if the USDHC2 instance 2 module is present in the SoC.
```

Comment:
Make this:
```
config HAS_MCUX_USDHC
	bool
	help
	  Set if the USDHC module is present in the SoC.
```

Reply:
I am still using "HAS_MCUX_USDHC1" and "HAS_MCUX_USDHC2", Some SoCs are supposed to have only one USDHC instance.
In subsys/disk/Kconfig, the dependency of DISK_ACCESS_USDHC is
"HAS_MCUX_USDHC1" or "HAS_MCUX_USDHC2" like this:
config DISK_ACCESS_USDHC
        bool "NXP i.MXRT USDHC driver"
        depends on (HAS_MCUX_USDHC1 || HAS_MCUX_USDHC2)